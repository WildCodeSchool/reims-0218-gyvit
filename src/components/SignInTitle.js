//TODO CREATE SIGN IN TITLE
import React from "react"

const SignInTitle = () => (
  <div>
    <h1>Sign it to Gyvit</h1>
    <p>Enter your details below </p>
  </div>
)

export default SignInTitle
