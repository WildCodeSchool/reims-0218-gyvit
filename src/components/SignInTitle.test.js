//TODO CREATE TEST

import React from "react"
import ReactDOM from "react-dom"
import renderer from "react-test-renderer"
import SignInTitle from "./SignInTitle"

it("renders without crashing", () => {
  const div = document.createElement("div")
  ReactDOM.render(<SignInTitle />, div)
  ReactDOM.unmountComponentAtNode(div)
})

it("renders link html", () => {
  const tree = renderer.create(<SignInTitle />).toJSON()
  expect(tree).toMatchSnapshot()
})
