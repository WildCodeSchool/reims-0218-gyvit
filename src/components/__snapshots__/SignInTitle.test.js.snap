// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders link html 1`] = `
<div>
  <h1>
    Sign it to Gyvit
  </h1>
  <p>
    Enter your details below 
  </p>
</div>
`;
